src/c++/perf_analyzer/genai-perf/genai_perf/utils.py

```diff
@@ -49,7 +53,22 @@ def load_yaml(filepath: Path) -> Dict[str, Any]:
 
 def load_json(filepath: Path) -> Dict[str, Any]:
     with open(str(filepath), encoding="utf-8", errors="ignore") as f:
-        return json.load(f)
+        content = f.read()
+        try:
+            return json.loads(content)
+        except json.JSONDecodeError:
+            snippet = content[:200] + ("..." if len(content) > 200 else "")
+            logger.error("Failed to parse JSON string: '%s'", snippet)
+            raise
```

Comment:
I think we can call `load_json_str` inside `load_json` to avoid duplicate blocks.
```suggestion
        content = f.read()
        load_json_str(content)
```

Reply:
ah my bad. I think it might need to be
```python
return load_json_str(content)
```

Reply:
Thanks, fixed. Re-running CI to confirm all is working.